required/latex-lab/testfiles-block/blocks-theorem-04.luatex.tlg

```diff
@@ -339,7 +339,7 @@ Completed box being shipped out [1]
 .....\glue(\spaceskip) 0.24774 plus 1.53333 minus 1.0222
 .....\pdfliteral page <lua data reference ...>
 .....\pdfliteral page <lua data reference ...>
-.....\pdfstartlink(*+*)x* attr{/Contents (ref) /Border [0 0 1] /H /I /C [0.701176 0.4 0.414118] /StructParent 1 } action struct name{thm.0.1} goto name{thm.0.1}
+.....\pdfstartlink(*+*)x* attr{/Contents <FEFF0047006F00200074006F002000640065007300740069006E006100740069006F006E002000740068006D002E0030002E0031> /Border [0 0 1] /H /I /C [0.701176 0.4 0.414118] /StructParent 1 } action struct name{thm.0.1} goto name{thm.0.1}
```

Comment:
Not sure what this is about: what needs to be accessible but is not?

Reply:
@car222222  I guess Frank would rather see `Go to destination thm.0.1` than `FEFF0047006F00200074006F002000640065007300740069006E006100740069006F006E002000740068006D002E0030002E0031`  (i'm sure you can read either with equal ease)

Reply:
thm.0.1 is probably considerably more useful, if a strange label;
but I do not think this has a lot to do with accessibility?

Reply:
The contents entry in itself is needed for a better tooltip (that came from one of the remarks from Peter when reviewing the wtpdf example). Using hex notation is not fully necessary, but if I use e.g. something like `grüßeご挨拶` in the title then I can convert at https://r12a.github.io/app-conversion/ and so check the hex notation. That is difficult with something like this (at the top the hyperref output, below the output I would get from str_convert):

~~~~
\376\377\000g\000r\000\374\000\337\000e\060\124\143\050\142\366
\376\377gr\303\274\303\237e\343\201\224\346\214\250\346\213\266
~~~~

Reply:
not sure octals are easier to read :-).  Well it was more a side remark that it would be nice if there is a way to easily read that kind of data in a \showoutput situation, but I understand that is likely to be impossible.